apps/server/src/config/swagger.json

```diff
@@ -337,7 +337,7 @@
 				"in": "query",
 				"required": false,
 				"schema": {
-					"type": "string"
+					"type": "boolean"
```

Comment:
what the what? `schema: true`?
explain, for my own edification?

Reply:
This just changes the input type in the swagger UI for the query parameter `references`